services/watson-extension/src/watson_extension/run.py

```diff
@@ -18,8 +19,8 @@
 # Connecting private routes (/)
 private_root.register_blueprint(health.blueprint)
 
-
 # Connect public routes ({config.base_url})
+public_root.register_blueprint(insights.blueprint)
```

Comment:
I missed the private vs public routes on the last pr. What's the plan for those? Is it
- public - accessible from internal.console
- private - accessible from main api? (confused about this part)

Reply:
private are stuff that we don't want to share in our /api/virtual-assistant/v2/* api - like health checks and eventually something else if required. Probably only the health atm

Reply:
Ah okay, sounds good!